tdms/matlabio/matlabio.cpp

```diff
@@ -1,164 +1,13 @@
-
-  */
 
-void updateMatlab3DArray(double *mArray, double arrayVal, int i, int j, int k, int nrows, int ncols){
-	*(mArray +k*nrows*ncols+ j*nrows + i) = arrayVal;
-}
-
-/* Gets an element from the 3-dimensional matlab array. This ensures no mistakes are made when indexing the matlab array.
-
-  mArray - pointer to the memory used to store the matlab array
-  i - row number of value to update
-  j - column number of value to update
-  k - index of third dimension
-  nrows - the number of rows in the matrix. This is equivalent to the number of
-          x elements under my standard of storing the xy data.
-  ncols - the number of coumns in the matrix. This is equivalent to the number of 
-          elements in the y direction under my standard of storing xy data
-
-  */
-
-double getMatlab3DArray(double *mArray, int i, int j, int k, int nrows, int ncols){
-	return *(mArray +k*nrows*ncols+ j*nrows + i);
-}
-
-#ifdef MIOFLAG
-void MatlabDisplayDouble(char *desc,double x){
-	mxArray *prhs[2];
-	char buf[50];
-	sprintf(buf, "%s: %.5f\n",desc,x);
-	prhs[0] = mxCreateDoubleMatrix(1,1, mxREAL);
-	*mxGetPr(prhs[0]) = 1;
-	prhs[1] = mxCreateString(buf);
-	mexCallMATLAB(0, NULL, 2, prhs ,"fprintf");
+using namespace std;
 
-}
-#endif
-#ifdef MIOFLAG
-void MatlabDisplayComplex(char *desc,complex<double> x){
-	mxArray *prhs[2];
-	char buf[50];
-	sprintf(buf, "%s: %f+i%f\n",desc,real(x),imag(x));
-	prhs[0] = mxCreateDoubleMatrix(1,1, mxREAL);
-	*mxGetPr(prhs[0]) = 1;
-	prhs[1] = mxCreateString(buf);
-	mexCallMATLAB(0, NULL, 2, prhs ,"fprintf");
-}
-#endif
-#ifdef MIOFLAG
-void MatlabDisplayString(char *desc,char *outstring){
-	mxArray *prhs[2];
-	char buf[50];
-	sprintf(buf, "%s: %s\n",desc,outstring);
-	prhs[0] = mxCreateDoubleMatrix(1,1, mxREAL);
-	*mxGetPr(prhs[0]) = 1;
-	prhs[1] = mxCreateString(buf);
-	mexCallMATLAB(0, NULL, 2, prhs ,"fprintf");
-}
-#endif
-#ifdef MIOFLAG
-void MatlabDisplayString(char *desc,char outchar){
-	mxArray *prhs[2];
-	char buf[50];
-	sprintf(buf, "%s: %c\n",desc,outchar);
-	prhs[0] = mxCreateDoubleMatrix(1,1, mxREAL);
-	*mxGetPr(prhs[0]) = 1;
-	prhs[1] = mxCreateString(buf);
-	mexCallMATLAB(0, NULL, 2, prhs ,"fprintf");
-}
-#endif
-/*
-  Checks whether or not to be verbose in reporting
-  
-  Returns 1 if it is necessary to report verbosely
-          0 if no verbose reporting
-  Just checks to see if a variable valled verbose is defined, if it is it returns 
-  1 if verbose is non zero or 0 if it is zero. If it is not defined, verbose output
-  is the default
-*/
-#ifdef MIOFLAG
-int ReportVerbosely(){
-	const mxArray *prhs;
-	prhs = mexGetVariablePtr("base","verbose");
-	if(prhs == NULL)
-	  prhs = mexGetVariablePtr( "caller","verbose");
-	if(prhs == NULL)
-	  prhs = mexGetVariablePtr( "global","verbose");
-	if(prhs == NULL){
-	  return 0;
-	}
-	else{
-		double *verb = mxGetPr(prhs);
-		if( *verb > 0)
-			return 1;
-		else
-			return 0;
-	}
-}
-#endif

```

Comment:
I completely approve of deleting unused stuff... why is it now unused?
(Excuse the dumb question)

Reply:
it was never used 👻